mapit_gb/management/commands/mapit_UK_import_boundary_line.py

```diff
@@ -59,14 +59,11 @@ def handle_label(self, filename, **options):
             ons_code = feat['CODE'].value if feat['CODE'].value not in ('999999', '999999999') else None
             unit_id = str(feat['UNIT_ID'].value)
             area_code = feat['AREA_CODE'].value
-            if hasattr(control, 'patch_boundary_line'):
-                patch = control.patch_boundary_line(name, ons_code, area_code)
```

Comment:
2015-10 temporarily did before it was realised that didn't work if you wanted your first import to be 2015-10 as you couldn't use that control file as it relied on previous imports. So no, nothing uses it and I imagine in future it will be the same, any patches will have to be here in case it's the first import (or the control files are written to work in either scenario, as actually I have done for 2016-05!).